app/views/render_async/_render_async.html.erb

```diff
@@ -5,7 +5,7 @@
   <%= placeholder %>
 </<%= html_element_name %>>
 
-<% content_for :render_async do %>
+<% content_for local_assigns.has_key?(:content_for_name) ? content_for_name : :render_async do %>
```

Comment:
This part should always be true, right? So we don't need `local_assigns.has_key?(:content_for_name)` and just leave `content_for_name`

Reply:
I will remove this on my branch, I'm about to merge